client/src/pages/Contest/ContestForm/ContestForm.tsx

```diff
@@ -55,7 +67,8 @@ const ContestForm: React.FC = () => {
       tempArray.deadline = true;
     }
     setErrors(tempArray);
-    console.log('Contest Fields', contestFields);
+    console.log('Contest Fields', tempArray);
```

Comment:
FYI You should generally remove console logs that are used for testing before you push to a PR